tensorflow/compiler/xla/service/gpu/tests/gpu_codegen_test.cc

```diff
@@ -51,9 +51,15 @@ void GpuCodegenTest::CompileAndVerifyPtx(
   std::unique_ptr<Executable> executable =
       std::move(CompileToExecutable(std::move(hlo_module)).ValueOrDie());
   string ptx_str(static_cast<GpuExecutable*>(executable.get())->text());
+
+  // On the ROCM platform the "ptx" string is not populated for the compiled
+  // executable, and hence the "ptx_str" will be empty. So disabling the
+  // pattern check on the ROCm platform
+#if !defined(TENSORFLOW_USE_ROCM)
```

Comment:
Sorry, same comment as before, would it be possible to make it a runtime check?

Reply:
You even have a `is_built_with_rocm_` variable now

Reply:
will fix this in the next commit I push out